File: python/8 kyu/245.py
# https://www.codewars.com/kata/58cb43f4256836ed95000f97/train/python

# Passed

def find_difference(a, b):
    calculate_volume = lambda dimensions: dimensions[0] * dimensions[1] * dimensions[2]
    volume_difference = abs(calculate_volume(a) - calculate_volume(b))
    return volume_difference

output = find_difference([3, 2, 5], [1, 4, 4])
print(output)

File: nodejs/7 kyu/245.js
// https://www.codewars.com/kata/5890579a34a7d44f3b00009e/train/javascript

// Passed

function manipulate(num) {
  num = String(num);
  let idxToSplit = Math.floor(num.length / 2);
  let manipulatedNum = Number(
    num.slice(0, idxToSplit) + "0".repeat(num.length - idxToSplit),
  );
  return manipulatedNum;
}

const output = manipulate(192827764920);
console.log(output);